icssc/AntAlmanac
# Mobile navbars

## Summary
Added mobile support for the nav bar and calendar tool bar.
![mobile_nav](https://user-images.githubusercontent.com/29494270/131930270-2d84121f-8abf-4c3d-95d8-1beb99dc914c.gif)

When the screen is under 750px, the logo's words will disappear and all of the buttons except load/save will be hidden behind a hamburger menu.
When the screen is under 630px, all of the the calendar's toolbar buttons except "Finals", undo, and delete will disappear behind a horizontal triple dot menu.

Other changes
- Finals button was moved next to the schedule selector
- Changed both nav components to use react hooks

## Test Plan
- View the site in mobile view and verify that it looks 🔥 
- View the site in desktop view and verify that nothing got messed up

## Issues
Relates to #125 

## Future Followup 
Consider implementing [mobile figma design](https://www.figma.com/file/Qi4kX8RFaP3Nw6wIWqtEGh/AntAlmanac-Mobile) when we redesign the rest of the site.
